wpilibj/src/main/java/edu/wpi/first/wpilibj/Watchdog.java

```diff
@@ -46,11 +46,11 @@
   /**
    * Watchdog constructor.
    *
-   * @param timeout The watchdog's timeout in seconds with microsecond resolution.
+   * @param timeoutSeconds The watchdog's timeout in seconds with microsecond resolution.
```

Comment:
This seems fishy and conflicting. Is it seconds or microseconds?

Reply:
I don't see how it is conflicting. It says you pass in a floating point number representing seconds, and up to 6 decimal places will actually do something.

Reply:
I see what you're saying. Perhaps replace `resolution` with `precision`? I feel like it'll be clearer.

Reply:
Resolution is the correct term here. https://www.arlynscales.com/blog/the-difference-between-accuracy-precision-and-resolution/